src/gui/guiScrollContainer.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Minetest
+Copyright (C) 2013 celeron55, Perttu Ahola <celeron55@gmail.com>
```

Comment:
wrong copyright

Reply:
I'm not good at such things. Feel free to suggest something.

Reply:
>  Copyright (C) 2020 DS-Minetest